tcms/core/middleware.py

```diff
@@ -29,6 +30,26 @@ def process_request(self, request):
                       '<a href="%(admin_url)s">change it</a>') % {
                           'doc_url': doc_url,
                           'admin_url': reverse('admin:sites_site_change', args=[site.pk])
-                      }
+                          }
+                )
+            )
+
+
+class CheckUnappliedMigrationsMiddleware(MiddlewareMixin):
+    def process_request(self, request):
+        doc_url = "https://kiwitcms.readthedocs.io/en/latest/\
+installing_docker.html#initial-configuration-of-running-container"
```

Comment:
why not use multi-line string concatenation like it was before ? Also used on lines 49-50 below.

Reply:
I'm confused, I had changed based on this feedback. https://github.com/kiwitcms/Kiwi/pull/1752#discussion_r444497884